node/text-generation/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "text-generation",
```

Comment:
Maybe include the name `huggingface-*` in the name as well as the folder name?

Reply:
Changing to `x-with-hugging-face` to match all previous other demos